x-pack/plugins/alerting/server/saved_objects/migrations.ts

```diff
@@ -274,7 +304,7 @@ function restructureConnectorsThatSupportIncident(
             subAction: 'pushToService',
             subActionParams: {
               incident: {
```

Comment:
Same here regarding `externalId`?

Reply:
`externalId` is only use from Case. it is not possible to have it there